### DISCUSSION TITLE
0.4.22

### DISCUSSION THREAD

**Discussion by smacke** [smacke/subsync#171] [General]
This release includes miscellaneous sync improvements, compatibility fixes, and other bugfixes. It also includes functionality for using a new algorithm, called "golden section search", for inferring the framerate ratio between the reference and input subtitles. To use, pass the `--gss` flag -- it may help in some cases where subtitles are still out of sync, if the issue is that the scaling is different from the predefined ratios ffsubsync tries.

<hr /><em>This discussion was created from the release <a href='https://github.com/smacke/ffsubsync/releases/tag/0.4.22'>0.4.22</a>.</em>

**Comment by jessielw** [comment]
Glad to see it!

**Reply by linlikun** [reply]
Why not ass? Only Srt? 

I found that when calling ffmpeg, it has a high requirement for hard drive speed
The speed of subtitle adjustment significantly decreases when starting multiple tasks simultaneously

**Reply by smacke** [reply]
SSA / ASS are both supported. Please file a detailed issue on https://github.com/smacke/ffsubsync/issues if it appears otherwise.

**Comment by linlikun** [comment]
Unfriendly support for Chinese paths or multi-level paths, sometimes reporting errors!

**Reply by smacke** [reply]
Hi there, can you file a detailed issue at https://github.com/smacke/ffsubsync/issues that includes the exact steps you took, the expected behavior, and the actual behavior? In particular, please include the command entered and the output.
